### PR TITLE
SEO exercise

### PR DESCRIPTION
**Pull request by DkSanjed**
Exercise html seo

### CODE REVIEW

**Comment by DkSanjed** [comment]
https://googlechrome.github.io/lighthouse/viewer/?psiurl=https%3A%2F%2Fgithub.com%2FDkSanjed%2Fhtml-tutorial%2Fblob%2Ffeature%2Fhtml-form%2Fform.html&strategy=mobile&category=performance&category=accessibility&category=best-practices&category=seo&category=pwa&utm_source=lh-chrome-ext#performance

**Comment by DkSanjed** [comment]
image

File: form.html [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+                        by Steve Jobs
+                    </blockquote>
+                </header>
+                <section>
+                    <h6>Forms</h6>
+                    <form>
+                        <fieldset>
+                            <legend><strong>Please enter your contact details</strong></legend>
+                            <div>
+                                <label for="name"><strong>Your name (required):</strong> </label>
```
Comment (juancho11gm) [inline form.html:47]:
```suggestion
                                <label for="use_name"><strong>Your name (required):</strong> </label>
```

File: form.html [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+                                <input type="checkbox" id="fullStack" name="course3" value="Fullstack Developer">
+                                <label for="fullStack"> <strong>Fullstack Developer</strong></label>                    
+                            </div>
+                            <div>
+                                <input type="checkbox" id="other" name="course4" value="other">
+                                <label for="other"> <strong>Other</strong></label>                    
+                            </div>
+                        </fieldset>
+                        <div>
+                            <label for="textArea"><strong>Please enter a message (optional):</strong></label>
```
Comment (juancho11gm) [inline form.html:81]: `<strong>` defines important text. Remember the importance with SEO.

**Comment by juancho11gm** [comment]
I think that with the link that you posted you are analyzing the Pull Request webpage and not your form webpage.

**Comment by DkSanjed** [comment]
https://googlechrome.github.io/lighthouse/viewer/?psiurl=https%3A%2F%2Fdksanjed.github.io%2Fhtml-tutorial%2Fform.html&strategy=mobile&category=performance&category=accessibility&category=best-practices&category=seo&category=pwa&utm_source=lh-chrome-ext

**Comment by DkSanjed** [comment]
image

**Pull request merged**